pkg/coscheduling/coscheduling_test.go

```diff
@@ -460,6 +461,8 @@ func TestPermit(t *testing.T) {
 				if got := f.RunPermitPlugins(context.TODO(), nil, tt.pods[i], ""); got.Code() != tt.expected[i] {
 					t.Errorf("[%v] want %v, but got %v", i, tt.expected[i], got.Code())
 				}
+				// The current pod does not exist in the snapshot during this scheduling cycle.
+				fakeSharedLister.Assumed(tt.pods[i], "node")
```

Comment:
It can be simplfied to `tt.pods[i] = "node"`? If so, we can eliminate the Assume() function.

Reply:
Done.